docs/concepts/accounts/depositauth.md

```diff
@@ -94,6 +90,9 @@ After the DepositPreauth transaction has been processed, the authorized account
 
 Preauthorization has no effect on the other ways to send money to an account with DepositAuth enabled. See [Precise Semantics](#precise-semantics) for the exact rules.
 
+{% amendment-disclaimer name="DepositPreauth" /%}
```

Comment:
Move this disclaimer to the beginner of this section, line 78?

Reply:
As discussed, moving other disclaimers to the end.